t.sentinel.import/t.sentinel.import.html

```diff
@@ -2,6 +2,9 @@ <h2>DESCRIPTION</h2>
 
 <em>t.sentinel.import</em> is a GRASS GIS addon Python script to
 download and import the Sentinel-2 scenes and create a STRDS.
+<p>
+Since Sentinel-2 <a href="https://forum.step.esa.int/t/info-introduction-of-additional-radiometric-offset-in-pb04-00-products/35431">Processing Baseline 4.0.0</a>, a systematic offset has been introduced to all reflectance values.
+To account for this, the <b>offset</b> option allows the user to indicate how reflectance data should be corrected (e.g. -1000).
```

Comment:
```suggestion
Since Sentinel-2 <a href="https://forum.step.esa.int/t/info-introduction-of-additional-radiometric-offset-in-pb04-00-products/35431">Processing Baseline 4.0.0</a>, a systematic offset has been introduced
to all reflectance values.<br>
To account for this, the <b>offset</b> option allows the user to indicate how reflectance data should
be corrected (e.g. -1000).
```

(break long lines, see https://github.com/OSGeo/grass/blob/main/doc/development/style_guide.md#markup-style-guide - "More notes on markup")